### DISCUSSION TITLE
Make it more clear how to edit environment settings

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#102116] [Repositories]
### Select Topic Area

Product Feedback

### Body

Right now does an entry in a Repository's `Environment` setting look like this:
![grafik](https://github.com/community/community/assets/11576465/7c576947-08ba-4fd4-8923-5ef807df777a)

This is a somewhat bad design, as it doesn't make it clear that you can edit the existing environment option by clicking the name itself.
The only thing that sticks out is the delete icon, which

**Comment by gil-son** [comment]
I agree with you. If someone add a new icon to edit, it's can be useful
